packages/devtools_shared/lib/src/memory/leaks/constants.dart

```diff
@@ -3,4 +3,4 @@
 // found in the LICENSE file.
 
 /// Name of extension that enables leak tracking for applications.
-const leakTrackingExtensionName = 'ext.memory-leak-tracking';
+const memoryLeakTrackingExtensionName = 'ext.flutter.memory.leak-tracking';
```

Comment:
use camel case: `ext.flutter.memory.leakTracking`

Also I'd recommend `ext.flutter.memoryLeakTracking` instead of `ext.flutter.memory.leakTracking` to be consistent with the other service extensions registered in the flutter framework.